### PR TITLE
YesTime term

### PR DESCRIPTION
**Pull request by alhamesrami**
I considered YesTime (Lines 365-366-367) as the https://yestime.com/ company, let me if I'm right. If yes, the file is good to go

### CODE REVIEW

**Comment by spencerjunraid** [comment]
I saw another PR for this file regarding YesTime. Can I close this one @alhamesrami ?

**Comment by bergware** [comment]
This PR has incorrect translation of the time machine lines

File: translations.txt [hunk 1]
```diff
@@ -362,9 +362,9 @@ Yes, I want to do this=نعم، أريد القيام بذلك
 Yes=نعم
 yes=نعم
 Yes hidden=نعم مخفي
-Yes Time Machine=نعم آلة الزمن
-YesTime Machine hidden=نعم/آلة الزمن مخفي
-YesTime Machine=نعم/آلة الزمن
+Yes Time Machine=جهاز YesTime
```
Comment (alhamesrami) [inline translations.txt:365]: نعم آلة الزمن

File: translations.txt [hunk 2]
```diff
@@ -362,9 +362,9 @@ Yes, I want to do this=نعم، أريد القيام بذلك
...
+YesTime Machine hidden=جهاز YesTime مخفي
```
Comment (alhamesrami) [inline translations.txt:366]: نعم آلة الزمن مخفية

File: translations.txt [hunk 3]
```diff
@@ -362,9 +362,9 @@ Yes, I want to do this=نعم، أريد القيام بذلك
...
+YesTime Machine=جهاز YesTime
```
Comment (alhamesrami) [inline translations.txt:367]: نعم آلة الزمن

**Pull request merged**